js2-mode.el

```diff
@@ -2632,14 +2632,17 @@ so many of its properties will be nil.
                                                         object
                                                         in-pos
                                                         each-pos
-                                                        foreach-p forof-p
+                                                        await-pos
```

Comment:
await-pos is unused, isn't it? When printing, etc.

Reply:
Same could be said for `each-pos`, of course, but let's not worry about that.

Reply:
I can remove them both, I think they aren't necessary.  But maybe some 3rd party depends on it, who knows.

Reply:
Up to you.